src/components/learner-credit-management/data/utils.js

```diff
@@ -119,12 +119,27 @@ export const getBudgetStatus = (startDateStr, endDateStr) => {
   const endDate = new Date(endDateStr);
 
   if (currentDate < startDate) {
-    return BUDGET_STATUSES.upcoming;
+    return {
+      status: BUDGET_STATUSES.upcoming,
+      variant: 'info',
```

Comment:
nit: perhaps `badgeVariant` just to be explicit about what `variant` its referring to.